indirect_value.h

```diff
@@ -114,6 +115,13 @@ class indirect_value : private indirect_value_base<T, C> {
 template <class T>
 indirect_value(T*) -> indirect_value<T>;
 
+template <class T, class C, class D>
+std::enable_if_t<std::is_swappable_v<C> && std::is_swappable_v<C>> swap(
```

Comment:
I think we want non-member swap as a hidden friend as before.

is there compelling reason to not do so?

Reply:
I thought that each type from the STL, if it provides a swap-function, it provides it as non-friend. But it turns out that I was wrong. The types stop_source, stop_token, jthread and [node-handle](http://eel.is/c++draft/container.node) provide their non-member swap function as hidden friend. So I guess, my argument "provide swap a non-friend due to consistency with STL" doesn't hold. I will turn it into a hidden friend as before.